exercises/03.schema-validation/02.problem.conform-action/README.mdx

```diff
@@ -15,41 +15,34 @@ import { json, redirect } from '@remix-run/node'
 import { z } from 'zod'
 
 const LoginSchema = z.object({
-	email: z.string().min(1, 'Email is required').email('Email is invalid'),
-	password: z.string().min(1, 'Password is required'),
+	email: z.string({ required_error: 'Email is required' }).email('Email is invalid'),
+	password: z.string({ required_error: 'Password is required' }),
```

Comment:
Conform now strips empty string to `undefined` to utilize zod required validation and there is no need to set `min(1)` anymore. Should I apply the same change to the `NoteEditorSchema`?

```tsx
// Instead of this
const NoteEditorSchema = z.object({
	title: z.string().min(titleMinLength).max(titleMaxLength),
	content: z.string().min(contentMinLength).max(contentMaxLength),
})

// Change it to this?
const NoteEditorSchema = z.object({
	title: z.string().max(titleMaxLength),
	content: z.string().max(contentMaxLength),
})
```

Reply:
Yes please!